spec/postgres_views/eod_assignment_count_spec.rb

```diff
@@ -0,0 +1,113 @@
+      { "day" => 2.days.ago.to_date, "assignable" => 1, "assigned" => 0 },
+      { "day" => 1.day.ago.to_date, "assignable" => 1, "assigned" => 0 },
+      { "day" => 0.days.ago.to_date, "assignable" => 1, "assigned" => 0 },
+    ])
+  end
+
+  it "assignable with decision, without assignment" do
+    create(
+      :event_submission,
+      events: [
+        { id: 1, event_type: "new_version", created_at: singuarlity - 70.hours },
+        { id: 2, event_type: "decision", created_at: singuarlity - 40.hours },
+      ],
+    )
+    expect(klass.where(day: period).map(&:attributes)).to eq([
+      { "day" => 2.days.ago.to_date, "assignable" => 1, "assigned" => 0 },
+      { "day" => 1.day.ago.to_date, "assignable" => 0, "assigned" => 0 },
+      { "day" => 0.days.ago.to_date, "assignable" => 0, "assigned" => 0 },
+    ])
+  end
+
+  it "assignable with assignment and unassignment" do
+    create(
+      :event_submission,
+      events: [
+        { id: 1, event_type: "new_version", created_at: singuarlity - 70.hours },
+        { id: 2, event_type: "assignment", created_at: singuarlity - 65.hours },
+        { id: 3, event_type: "unassignment", created_at: singuarlity - 40.hours },
+      ],
+    )
+    expect(klass.where(day: period).map(&:attributes)).to eq([
+      { "day" => 2.days.ago.to_date, "assignable" => 1, "assigned" => 1 },
+      { "day" => 1.day.ago.to_date, "assignable" => 1, "assigned" => 0 },
+      { "day" => 0.days.ago.to_date, "assignable" => 1, "assigned" => 0 },
+    ])
+  end
+
+  it "assignable with multiple assignment and unassignment" do
+    create(
+      :event_submission,
+      events: [
+        { id: 1, event_type: "new_version", created_at: singuarlity - 70.hours },
+        { id: 2, event_type: "assignment", created_at: singuarlity - 65.hours },
+        { id: 3, event_type: "unassignment", created_at: singuarlity - 40.hours },
+        { id: 4, event_type: "assignment", created_at: singuarlity - 35.hours },
+      ],
+    )
+    expect(klass.where(day: period).map(&:attributes)).to eq([
+      { "day" => 2.days.ago.to_date, "assignable" => 1, "assigned" => 1 },
+      { "day" => 1.day.ago.to_date, "assignable" => 1, "assigned" => 1 },
+      { "day" => 0.days.ago.to_date, "assignable" => 1, "assigned" => 1 },
+    ])
+  end
+
+  it "multiple assignable (single application)" do
+    create(
+      :event_submission,
+      events: [
+        { id: 1, event_type: "new_version", created_at: singuarlity - 70.hours },
+        { id: 2, event_type: "assignment", created_at: singuarlity - 65.hours },
+        { id: 3, event_type: "sent_back", created_at: singuarlity - 60.hours },
+        { id: 4, event_type: "provider_updated", created_at: singuarlity - 45.hours },
+        { id: 5, event_type: "assignment", created_at: singuarlity - 20.hours },
+      ],
+    )
+    expect(klass.where(day: period).map(&:attributes)).to eq([
+      { "day" => 2.days.ago.to_date, "assignable" => 0, "assigned" => 0 },
```

Comment:
I guess this is correct as the application has been `sent_back` to the provider so isn't assigned or unassigned at the end of this day????